lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -295,9 +295,10 @@ sub validate_syntax {
         }
 
         if ( defined $syntax_input->{profile} ) {
-            my @profiles = map lc, $self->{config}->ListPublicProfiles();
-            return { status => 'nok', message => encode_entities( "Invalid profile option format" ) }
-            unless ( grep { $_ eq lc $syntax_input->{profile} } @profiles );
+            my %profiles = ( $self->{config}->PUBLIC_PROFILES, $self->{config}->PRIVATE_PROFILES );
+            if ( !exists $profiles{ $syntax_input->{profile} } ) {
```

Comment:
if the profile name is normalized in the config shouldn't the profile parameter be normalized as well?

Reply:
Your right. I fixed it.